metabase/metabase
# feat(sdk): option to hide dashboard card title

Closes https://github.com/metabase/metabase/issues/43819

### Description

Adds the `withCardTitle` option to the `StaticDashboard` component which allows SDK users to hide the card title via `withCardTitle={false}`

### How to verify

- Navigate to the `test-card-title` branch in the demo app, and run the demo app
- Ensure that `routes/analytics/index.tsx` in the demo app has a `withCardTitle={false}` prop set in StaticDashboard
- Click on the Analytics Page. The card title should be hidden.

### Demo

![CleanShot 2567-06-10 at 23 24 18@2x](https://github.com/metabase/metabase/assets/4714175/09f0fa05-107f-4120-aa89-7d5ea54680fd)

### Checklist
